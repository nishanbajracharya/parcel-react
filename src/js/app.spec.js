import React from 'react';
import renderer from 'react-test-renderer';

import { App } from './app';

test('App should render a h1 tag', () => {
  const component = renderer.create(<App />).toJSON();

  expect(component).toMatchSnapshot();
});
