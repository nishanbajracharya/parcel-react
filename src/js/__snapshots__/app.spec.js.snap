// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render a h1 tag 1`] = `
<h1>
  Setting up React with Parcel bundler
</h1>
`;
